### PR TITLE
Add Required Tools section

### PR DESCRIPTION
**Pull request by NicolasHug**
See https://discord.com/channels/890339612441063494/1127658896710975518 for motivation and initial discussion.

This adds a short list of "additional stuff you will need". Some of them are really obvious, some of them aren't. I kept the list short, avoided links, and added a disclaimer to keep maintenance burden to a minimum.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -131,7 +130,7 @@ _Will enable brew and steam temperature control as well as pump control based on
 * [C-M4 screw K-Type thermocouple sensor](https://www.aliexpress.com/item/1005004948080451.html)
 * [40DA SSR Relay](https://www.aliexpress.com/item/4000045425145.html)
 * [Heat-resistant Silicone Wire](https://bit.ly/3tjSQbI)
-  * **18AWG - 1m:** Black, Red, White
+  * **18AWG - 2m:** Black, Red, White
```
Comment (NicolasHug) [inline docs/README.md:133]: I tool the liberty to update that, can remove if needed. I found 1m to be barely enough as I needed to use some of it to bypass the PCB. I feel like adding 1m won't hurt anyone, even those that don't need it.

File: docs/README.md [hunk 2]
```diff
@@ -266,6 +265,20 @@ _Will enable pump control based on active pressure feedback, this enables pressu
...
+<!-- tabs:end -->
+<!-- tab:Required tools -->
+> [!WARNING]
+> This is not necessarily an exhaustive list. You may need additional tools depending on your specific install. Please refrain from asking for references: use common sense, and search the Discord server for existing recommendations.
+
+* Soldering: iron, solder, flux, wick, third-hand tool
+* Multimeter - for continuity testing
+* Thermal paste - for thermocouple
+* Crimp pliers, wire stripper/cutter
+* Strip board or WAGO connectors - for lego build
```
Comment (NicolasHug) [inline docs/README.md:277]: Need someone to confirm: is the strip-board needed as well for the PCB build?

**Comment by NicolasHug** [comment]
Looks like there's no CI job that builds the docs on PRs (or is there?) so this is what it looks like:

image

**Comment by GAGGIUINO** [comment]
Hi - thanks for the PR, i don't disagree with a section on required tools but the whole docs will undergo a major refactoring to simplify a lot of the saga that's been accumulated aver the course of many PRs and streamline even more the install process for the uninitiated (hopefully though low key i know this is mission impossible as folks will always find shit they need to cry about).

So my point is that i won't be merging this as the risk is that it will generate more questions now than solve and take away focus from where priority is necessary.

**Pull request closed without merging**